### ISSUE TITLE
set proper input

### ISSUE THREAD

**Issue by mossishahi** [aquaskyline/SOAPdenovo2#46]
I'm trying to set output of Minia assembler(a contig file : contig.fa ) as input of SOAPdenovo-fusion to scaffold my contigs. 
As is searched in the net and also previous issues I understood that this line of code could be useful:
`SOAPdenovo-fusion -D -K 41 -c ./final.contigs.fa`
but unfortunately, the result does not match with my expectation to see a `.fa` file which is concluded by scaffolding contigs and is my genome. and only is some files about my data.
in the next step, I decided to try this line of code:
`SOAPdenovo-fusion -s ./config -p 8 -K 41 -g final.contigs -c  ./final.contigs.fa`
however, I don't know what the config file is when I don't have any other setting.

In summary, I need a command to set 1.my contig file  2. K size and get the result in `fa `format which is my genome.

I'm in an emergency
Thanks for any help.
